php7.0/Dockerfile

```diff
@@ -49,6 +51,9 @@ RUN curl \
 RUN mkdir -p /app && rm -fr /var/www/html && ln -s /app /var/www/html
 WORKDIR /app
 
+# Add Composer's vendor/bin/ to the PATH
+ENV PATH "$PATH:/app/vendor/bin"
```

Comment:
run.sh does not require this. This is only required to use modules installed by Composer from the command line.
Without this, I can install the phpunit package and write unit test files, but I cannot run those test using `phpunit` from the command line :-/